script.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/python3
+
+import psycopg2
+import requests
+from bs4 import BeautifulSoup
+
+
+def connect():
+    conn = psycopg2.connect(host='db', user="postgres", database="postgres-2", password='mmuntean1!')
+    cur = conn.cursor()
+    cur.execute("SELECT name, description, imdb_id FROM public.movies_movie WHERE movies_movie.description='';")
+    print("The number of parts: ", cur.rowcount)
+    row = cur.fetchone()
```

Comment:
I'm not entirely sure if the first row is not skipped because the first row assignment is not used.